src/Money.php

```diff
@@ -36,6 +38,11 @@
     private static $calculator;
 
     /**
+     * @var Currencies
+     */
+    private static $currencies;
```

Comment:
Yes, initially I was thinking the same, the logic behind conversion for sure will get bigger and maybe it'll be better to move it, but after looking a while on the Money class and seeing calculator instances are also holded by static property I decided to go the samy way to keep the class organization.